packages/muelu/doc/TutorialReadTheDocs/pages/xml_interface_for_advanced_users.rst

```diff
@@ -0,0 +1,215 @@
+      	</ParameterList>
+    	</ParameterList>
+    	
+.. admonition:: Exercise 5
+
+	Pick out one level smoother from above and use them for your problem. Note that you may have to adapt the **relaxation: damping factor** for reasonable results.
+	
+Level smoothers for the Epetra and Tpetra stack
+-----------------------------------------------
+Generally, the **TrilinosSmoother** factory class tries to provide the same set of standard level smoothers for the **Epetra** and **Tpetra** stack as far as possible. Under the hood it uses **Ifpack** and **Ifpack2**.
+**MueLu** partially tries to internally translate the parameters from **Ifpack2** to **Ifpack**, but this is not always possible and error-prone. The available values for the **type** parameter are **RELAXATION**, **CHEBYSHEV**, **ILUT**, **RILUK** and **ILU**.
+
+.. note::
+
+	In order to define a multigrid hierarchy that is working for both the **Epetra** and **Tpetra** stack it is recommended to use **CHEBYSHEV** or **RELAXATION** based smoothers.
+
+
+The **TrilinosSmoother** factory may provide support more types of smoothers, but these have to be considered as experimental and often are only available for the **Tpetra** branch. Please check the source code for a list of all available smoothers and options. More advanced smoothing strategies will also introduced later in this guide (e.g. :ref:`advanced_concepts/line-smoothing`).
+
+Advanced features
+=================
+
+MueLu allows full control over the behavior of the multigrid levels. Here, we demonstrate the capabilities of MueLu using the level smoothers.
+Take a look at the following example XML parameter list
+
+.. literalinclude:: ../../../test/tutorial/s2_adv_b.xml
+  :language: xml
+
+
+You can find the parameters in **../../../test/tutorial/s2_adv_b.xml**. We have one building block **BackwardGaussSeidel** representing the level smoother that we want to use in our multigrid hierarchy. As one can see from the **Hierarchy** block we request a 4 level multigrid method. There are two blocks called **Finest** and **Remaining** describing the behavior of the different multigrid levels. Note the **startLevel** parameter in the block **Remaining**. This parameter is missing in the **Finest** block (where it is assumed to be the default value which is zero). That is, in this example we use the backward Gauss-Seidel method as pre-smoother on the finest level (note the keyword **NoFactory** for **PostSmoother**). The parameter **startLevel=1** in the **Remaining** block means that for level 1 and all coarser levels (unless there is another block with **startLevel > 1**) the building blocks from **Remaining** shall be used for the multigrid setup.
+That is, on the multigrid levels 1 and 2 the backward Gauss-Seidel method is used for post smoothing only. The corresponding multigrid hierarchy has the form
+
+.. warning:: 
+  
+	\printScreenOutput{s2_adv_b.txt_3.fragment_3.fragment}
+	-> Auch in Exercise 6 ersetzen!
```

Comment:
I saw this when I was doing a quick pass for typos last night. I don't know if the warnings appear for the user or not, but if the user doesn't see this then it should be fine 😛

Reply:
I don't know, if and how the warnings are printed. But I can translate that sentence: "Auch in Exercise 6 ersetzen!" --> Replace also in Exercise 6. 😉